luaui/Widgets/gui_nanotrans_range.lua

```diff
@@ -0,0 +1,97 @@
+function widget:GetInfo()
+    return {
+       name         = "Nano range on transport",
+       desc         = "Draw a circle around a transport carrying a nano when its about to unload",
+       author       = "Cheva",
+       date         = "December 2024",
+       license      = "GNU GPL, v2 or later",
+       layer        = 0,
+       enabled      = true
+    }
+end
+
+--------------------------------------------------------------------------------
+--vars
+--------------------------------------------------------------------------------
+local circleDivs = 96
+local range
+local isNanoTC = {}
```

Comment:
```suggestion
local isImmobileBuilder = {}
```
Or `isConstructionTurret` or something similar. For code clarity, avoid using abbreviations, short forms, and slang.